src/App.js

```diff
@@ -2,78 +2,133 @@
 import React, { useState, useEffect } from "react"; //import useState
 import "./App.css";
 import BottomRow from "./BottomRow";
-import HomeTouchBtn from "./components/homeTouchBtn";
+import soundFile from "./assets/air-horn-club-sample_1.mp3";
 
 function App() {
   //TODO: STEP 2 - Establish your applictaion's state with some useState hooks.  You'll need one for the home score and another for the away score.
 
   //home
-  const [homeTeam, setHome] = useState("Lions");
+  const [homeTeam, setHome] = useState("Raiders");
   const [homeScore, setHomeScore] = useState(0);
   //away
-  const [awayTeam, setAway] = useState("Tigers");
+  const [awayTeam, setAway] = useState("Bears");
   const [awayScore, setAwayScore] = useState(0);
 
+  //quarter
+  const [quarter, setQuarter] = useState(1);
+
   //timer
-  function Timer() {
-    const [time, setTimer] = useState("30:00");
-  }
+  const gameTime = 30;
+  const [time, setTimer] = useState(gameTime);
+  const [timeOn, setTimeOn] = useState(false);
+  const [playedSound, setPlayedSound] = useState(false);
+
+  const [playSound, setPlaySound] = useState(0);
+
+  const quarterFunc = () => {
+    if (quarter < 4) {
+      setQuarter(quarter + 1);
+    } else setQuarter(1);
+  };
+
+  const toggle = () => {
+    setTimeOn(!timeOn);
+  };
+
+  useEffect(() => {
+    let interval = null;
+    if (timeOn && time >= 1) {
+      interval = setInterval(() => {
+        setTimer(seconds => seconds - 1);
+      }, 1000);
+    } else if (!timeOn && time !== 0) {
+      clearInterval(interval);
+    } else if (time === 0) {
+      if (!playedSound) {
+        setPlayedSound(true);
+        setPlaySound(!playSound);
+        toggle();
+        setTimeout(() => {
+          setTimer(gameTime);
+        }, 1000);
+      }
+    }
+    return () => {
+      clearInterval(interval);
+    };
+  }, [timeOn, time]);
 
   return (
-    <div className="container">
-      <section className="scoreboard">
-        <div className="topRow">
-          <div className="home">
-            <h2 className="home__name">{homeTeam}</h2>
+    <div className='container'>
+      <section className='scoreboard'>
+        <h2 className='time_name'>Time</h2>
+        <div className='topRow'>
+          <div className='home'>
+            <h2 className='home__name'>{homeTeam}</h2>
 
             {/* TODO STEP 3 - We need to change the hardcoded values in these divs to accept dynamic values from our state. */}
 
-            <div className="home__score">{homeScore}</div>
+            <div className='home__score'>{homeScore}</div>
+          </div>
+
+          <div className='time'>
+            <div className='timer'>{time}</div>
+            {/* <div className='timer'>{time}</div>
+            <div className='timer'>{time}</div>
+            <div className='timer'>{time}</div> */}
```

Comment:
I would recommend removing commented out code before committing. This is considered Zombie code.
Git's job is to remember your old code so you shouldn't ever need to comment out code in a commit if you are committing properly.